app/models/manageiq/providers/hawkular/middleware_manager/middleware_server.rb

```diff
@@ -1,5 +1,9 @@
 module ManageIQ::Providers
   class Hawkular::MiddlewareManager::MiddlewareServer < MiddlewareServer
+    def feed
+      CGI.unescape(super)
```

Comment:
@himdel  I'd say that the factory is wrong. No server is added by the provider without a feed. May be also the database column is wrongly defined and should be "not null".